src/vector/rageshakesetup.js

```diff
@@ -44,3 +45,17 @@ function initRageshake() {
 }
 
 initRageshake();
+
+global.mxSendRageshake = function(text, withLogs) {
+    require(['matrix-react-sdk/lib/rageshake/submit-rageshake'], (s) => {
+        s(SdkConfig.get().bug_report_endpoint_url, {
+            userText: text,
+            sendLogs: withLogs,
+            progressCallback: console.log,
```

Comment:
Using just `console.log` as a callback breaks in one of the browsers (I forget which one) because it expects `console.log` to be called as a method on the `console` object, rather than just a function.  It can be fixed by using `console.log.bind(console)` a the callback instead.